src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/runner/InitialStateRunner.kt

```diff
@@ -37,12 +39,19 @@ class InitialStateRunner(
     }
 
     private fun setupWebhook() {
-        runCatching {
-            webhookService.registerWebhook()
-        }.onSuccess {
-            logger.info { "webhook registered successfully" }
-        }.onFailure {
-            logger.error(it) { "webhook registration failed" }
+        if (gitlabWebhookUrl.isNotBlank() && gitlabWebhookUrl.isNotEmpty()) {
```

Comment:
Isn't `isNotBlank()` sufficient? Why do we need to check also for non empty?

Reply:
you're right, isNotBlank should be enough. I will push a fix.

Reply:
fixed in the latest commit.